app/src/main/java/bg/zahov/app/ui/measures/MeasuresScreen.kt

```diff
@@ -0,0 +1,68 @@
+package bg.zahov.app.ui.measures
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import bg.zahov.app.ui.custom.CommonDivider
+
+@Composable
+fun MeasuresScreen(navigateInfo: (String) -> Unit, viewModel: MeasuresViewModel = viewModel()) {
+    MeasuresContent(
+        items = viewModel.getAllMeasures(),
```

Comment:
I know it's just a list with enums but create a state for consistency and separation of responsibilities